features/resources/IFC2X3/pset_definitions.csv

```diff
@@ -7,10 +7,10 @@ Pset_ActuatorTypeHydraulicActuator,['IfcActuatorType'],IfcActuatorType.Predefine
ance': 'PEnum_AirSideSystemType', 'values': ['CONSTANTVOLUME', 'CONSTANTVOLUMESINGLEZONE', 'CONSTANTVOLUMEMULTIPLEZONEREHEAT', 'CONSTANTVOLUMEBYPASS', 'VARIABLEAIRVOLUME', 'VARIABLEAIRVOLUMEREHEAT', 'VARIABLEAIRVOLUMEINDUCTION', 'VARIABLEAIRVOLUMEFANPOWERED', 'VARIABLEAIRVOLUMEDUALCONDUIT', 'VARIABLEAIRVOLUMEVARIABLEDIFFUSERS', 'VARIABLEAIRVOLUMEVARIABLETEMPERATURE', 'OTHER', 'NOTKNOWN', 'UNSET\n']}}, {'property_name': 'AirSideSystemDistributionType', 'property_type': 'IfcPropertyEnumeratedValue', 'data_type': {'instance': 'PEnum_AirSideSystemDistributionType', 'values': ['SINGLEDUCT', 'DUALDUCT', 'MULTIZONE', 'OTHER', 'NOTKNOWN', 'UNSET']}}, {'property_name': 'TotalAirflow', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcVolumetricFlowRateMeasure', 'unit_type': ['VOLUMETRICFLOWRATEUNIT']}}, {'property_name': 'EnergyGainTotal', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPowerMeasure', 'unit_type': ['POWERUNIT']}}, {'property_name': 'AirflowSensible', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcVolumetricFlowRateMeasure', 'unit_type': ['VOLUMETRICFLOWRATEUNIT']}}, {'property_name': 'EnergyGainSensible', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPowerMeasure', 'unit_type': ['POWERUNIT']}}, {'property_name': 'EnergyLoss', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPowerMeasure', 'unit_type': ['POWERUNIT']}}, {'property_name': 'LightingDiversity', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': IfcPositiveRatioMeasure, 'unit_type': 'None'}}, {'property_name': 'InfiltrationDiversitySummer', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': IfcPositiveRatioMeasure, 'unit_type': 'None'}}, {'property_name': 'InfiltrationDiversityWinter', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': IfcPositiveRatioMeasure, 'unit_type': 'None'}}, {'property_name': 'ApplianceDiversity', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': IfcPositiveRatioMeasure, 'unit_type': 'None'}}, {'property_name': 'LoadSafetyFactor', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': IfcPositiveRatioMeasure, 'unit_type': 'None'}}, {'property_name': 'HeatingTemperatureDelta', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcThermodynamicTemperatureMeasure', 'unit_type': ['THERMODYNAMICTEMPERATUREUNIT']}}, {'property_name': 'CoolingTemperatureDelta', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcThermodynamicTemperatureMeasure', 'unit_type': ['THERMODYNAMICTEMPERATUREUNIT']}}, {'property_name': 'Ventilation', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcVolumetricFlowRateMeasure', 'unit_type': ['VOLUMETRICFLOWRATEUNIT']}}, {'property_name': 'FanPower', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPowerMeasure', 'unit_type': ['POWERUNIT']}}]"
```

Comment:
```suggestion
Pset_AirSideSystemInformation,"['IfcSpatialStructureElement', 'IfcSystem']",,"[{'property_name': 'Name', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcLabel', 'unit_type': 'None'}}, {'property_name': 'Description', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcLabel', 'unit_type': 'None'}}, {'property_name': 'AirSideSystemType', 'property_type': 'IfcPropertyEnumeratedValue', 'data_type': {'instance': 'PEnum_AirSideSystemType', 'values': ['CONSTANTVOLUME', 'CONSTANTVOLUMESINGLEZONE', 'CONSTANTVOLUMEMULTIPLEZONEREHEAT', 'CONSTANTVOLUMEBYPASS', 'VARIABLEAIRVOLUME', 'VARIABLEAIRVOLUMEREHEAT', 'VARIABLEAIRVOLUMEINDUCTION', 'VARIABLEAIRVOLUMEFANPOWERED', 'VARIABLEAIRVOLUMEDUALCONDUIT', 'VARIABLEAIRVOLUMEVARIABLEDIFFUSERS', 'VARIABLEAIRVOLUMEVARIABLETEMPERATURE', 'OTHER', 'NOTKNOWN', 'UNSET\n']}}, {'property_name': 'AirSideSystemDistributionType', 'property_type': 'IfcPropertyEnumeratedValue', 'data_type': {'instance': 'PEnum_AirSideSystemDistributionType', 'values': ['SINGLEDUCT', 'DUALDUCT', 'MULTIZONE', 'OTHER', 'NOTKNOWN', 'UNSET']}}, {'property_name': 'TotalAirflow', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcVolumetricFlowRateMeasure', 'unit_type': ['VOLUMETRICFLOWRATEUNIT']}}, {'property_name': 'EnergyGainTotal', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPowerMeasure', 'unit_type': ['POWERUNIT']}}, {'property_name': 'AirflowSensible', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcVolumetricFlowRateMeasure', 'unit_type': ['VOLUMETRICFLOWRATEUNIT']}}, {'property_name': 'EnergyGainSensible', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPowerMeasure', 'unit_type': ['POWERUNIT']}}, {'property_name': 'EnergyLoss', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPowerMeasure', 'unit_type': ['POWERUNIT']}}, {'property_name': 'LightingDiversity', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPositiveRatioMeasure', 'unit_type': 'None'}}, {'property_name': 'InfiltrationDiversitySummer', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPositiveRatioMeasure', 'unit_type': 'None'}}, {'property_name': 'InfiltrationDiversityWinter', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPositiveRatioMeasure', 'unit_type': 'None'}}, {'property_name': 'ApplianceDiversity', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPositiveRatioMeasure', 'unit_type': 'None'}}, {'property_name': 'LoadSafetyFactor', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPositiveRatioMeasure', 'unit_type': 'None'}}, {'property_name': 'HeatingTemperatureDelta', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcThermodynamicTemperatureMeasure', 'unit_type': ['THERMODYNAMICTEMPERATUREUNIT']}}, {'property_name': 'CoolingTemperatureDelta', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcThermodynamicTemperatureMeasure', 'unit_type': ['THERMODYNAMICTEMPERATUREUNIT']}}, {'property_name': 'Ventilation', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcVolumetricFlowRateMeasure', 'unit_type': ['VOLUMETRICFLOWRATEUNIT']}}, {'property_name': 'FanPower', 'property_type': 'IfcPropertySingleValue', 'data_type': {'instance': 'IfcPowerMeasure', 'unit_type': ['POWERUNIT']}}]"
```

There are 3-4 unquoted values for the "instance" keys under "data_type" for properties such as ApplianceDiversity and LightingDiversity that are causing JSON Decode errors on the model data provided with the bug report.  The suggested change should fix this.